dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -417,6 +435,22 @@ Resources:
     DependsOn:
       - VpcEndpoint
       - {{ service.prefix }}DBWriterInstance
+      - AdditionalNatGateway1
+      - AdditionalNatGateway2
+      - AdditionalPrivateRoute1
+      - AdditionalPrivateRouteTable1
+      - AdditionalPrivateRouteTable1Association
+      - AdditionalPrivateRoute2
+      - AdditionalPrivateRouteTable2
+      - AdditionalPrivateRouteTable2Association
+      - {{ service.prefix }}SecretAuroraClusterAttachment
+      - {{ service.prefix }}EnvironmentIngress
+      - {{ service.prefix }}SecretsManagerIngress
+      - {{ service.prefix }}LambdaIngress
+      - {{ service.prefix }}SecretsManagerEgress
+      - {{ service.prefix }}LambdaEgress
+      - {{ service.prefix }}HTTPSEgress
+      - {{ service.prefix }}KeyAlias
```

Comment:
Wow, that's quite the list!

Reply:
This is probably the spike ticket which we already have: https://uktrade.atlassian.net/browse/DBTP-476